csrankings-a.csv

```diff
@@ -412,6 +412,7 @@ Akira Imakura,University of Tsukuba,https://researchmap.jp/imakura?lang=en,wPEcf
 Akira Kawaguchi,CUNY,https://www.ccny.cuny.edu/profiles/akira-kawaguchi,7cF0q3YAAAAJ
 Akira Nukada,University of Tsukuba,https://www.hpcs.cs.tsukuba.ac.jp/~nukada,vjE9JnYAAAAJ
 Akiya Kamimura,University of Tsukuba,https://staff.aist.go.jp/kamimura.a/dsysdl/index.html,6NFwDKEAAAAJ
+Akond Ashfaque Ur Rahman,Auburn University,https://akondrahman.github.io,6yZrePQAAAAJ

```

Comment:
Add an entry for each DBLP alias (see https://dblp.org/pid/06/10426.html).

Reply:
I will make the necessary changes and create another pull request. Thanks for the feedback.